### ISSUE TITLE
Color of the message should be obtained from SQ QualityGate

### ISSUE THREAD

**Issue by erichirlemann** [dbac2002/sonar-hipchat-plugin#3]
The color of the hipchat message is computed from custom code.
By matter of consistency, it would be great to have it obtained from SQ QualityGate.

**Comment by dbac2002** [comment]
yes you are right - it should get the color from the quality gate, if possible - i have yet to find out how to retrieve this information from the api
